cmd/rigs/cmd/publish_layers.go

```diff
@@ -36,16 +36,17 @@ var layersCmd = &cobra.Command{
 		chunksDir := viper.GetString("chunks-dir")
 
 		if chunksDir == "" && !c.Defined() {
-			c, err = dirPublisher.DirToIpfs(ctx, path, "layers")
+			c, err = pub.DirToIpfs(ctx, path)
 			checkErr(err)
 			fmt.Printf("Images added to IPFS with cid %s\n", c.String())
+			checkErr(localStore.TrackCid(ctx, "layers", c.String()))
 		}
 		if chunksDir == "" && c.Defined() {
-			chunksDir, err = dirPublisher.CidToCarChunks(ctx, c)
+			chunksDir, err = pub.CidToCarChunks(ctx, c)
 			checkErr(err)
 			fmt.Printf("Car chunks written to folder %s\n", chunksDir)
 		}
-		checkErr(dirPublisher.CarChunksToNftStorage(
+		checkErr(pub.CarChunksToCarStorage(
```

Comment:
These changes and similar changes elsewhere are just to accommodate the refactor that abstracts interaction with web3.storage and nft.storage with an interface.